### DISCUSSION TITLE
Phononic thermal conductivity from OpenMX

### DISCUSSION THREAD

**Discussion by LauraCaputo** [ttadano/alamode#143] [Q&A] [answered]
Dear all,

I am quite new to this topic and I am trying to learn how to properly do calculations on the Figure of Merit of some periodic models.

In particular, due to their periodicity and the zero voltage applied, I am doing Equilibrium calculations as reported [here](https://www.openmx-square.org/openmx_man3.8/node122.html).

After calculating the Transmittance, Seeback coefficient and electronic thermal conductivity, I would like to conclude my study by including also the phononic thermal conductivity. Reading from the manual, something is not quite clear to me: in my calculations, I include two electrodes and a central region, that are equal due to the periodicity. When moving to the calculation of phononic properties, should I still consider the 'full' model with two electrodes? Only the central region is enough?

Thank you!

**Comment by ttadano** [comment, marked as answer]
The setup described in that manual page is for simulating conductance using NEGF. ALAMODE does not support simulating phonon conductance based on NEGF; it only supports calculating phononic conductivity in the diffusive regime. You cannot mix electronic *conductance* (device shape dependent) and thermal *conductivity* (bulk property) for evaluating the figure of merit.

**Reply by LauraCaputo** [reply]
Thanks a lot.
Can I kindly ask you if you have any suggestions on which codes can be used correctly and coherently to calculate the figure of merit?

**Reply by ttadano** [reply]
I'm not very familiar with NEGF codes. Please search them, for example, at https://ma.issp.u-tokyo.ac.jp/en.

**Reply by LauraCaputo** [reply]
Thanks again. So if I understand correctly, I could interface ALAMODE with VASP, using a periodic cell (no electrodes and no NEGF) to calculate the phonon thermal conductivity.
This could be mixed with other bulk properties calculated with other codes (like Boltztrap) to finally calculate the figure of merit.
Am I correct?

**Reply by ttadano** [reply]
Yes. I usually use Boltztrap or Perturbo to compute carrier conductivity and Seebeck coefficient and use ALAMODE to compute lattice thermal conductivity.

**Reply by LauraCaputo** [reply]
Ok, thanks a lot.
One final question: I have read that BoltzTraP calculated the electronic part of the thermal conductivity in function of tau (Ke/tau).
When comparing the values with the ones of ALAMODE, should one multiply the electronic part with a certain value of tau or is this directly summable with Kp calculated by Alamode?

**Reply by ttadano** [reply]
You will have to estimate Ke, instead of Ke/tau, by assuming a reasonable value for electron tau. There are many papers going beyond this level of approximation (estimate tau using more realistic model), so please check them.

**Reply by LauraCaputo** [reply]
Ok, thanks!

**Reply by LauraCaputo** [reply]
Sorry again for writing you here. I have been digging in the Perturbo documentation, since you mentioned it.
I see that in this case, one could actually calculate Ke instead of Ke/tau and one shouldn't approximate on a fixed constant value of tau. This would make it easier to interface directly with Alamode for the calculation of KT. Is it right?
